GitUI/UserControls/BlameControl.cs

```diff
@@ -388,6 +435,48 @@ private void BuildAuthorLine(GitBlameLine line, StringBuilder lineBuilder, strin
             }
         }
 
+        private IList<Color> GetGradients()
+        {
+            // Color chosen from: https://colorbrewer2.org/#type=sequential&scheme=Greens&n=9
+            return new List<Color>
+            {
+                Color.FromArgb(247, 252, 245).AdaptBackColor(),
+                Color.FromArgb(229, 245, 224).AdaptBackColor(),
+                Color.FromArgb(199, 233, 192).AdaptBackColor(),
+                Color.FromArgb(161, 217, 155).AdaptBackColor(),
+                Color.FromArgb(116, 196, 118).AdaptBackColor(),
+                Color.FromArgb(65, 171, 93).AdaptBackColor(),
+                Color.FromArgb(35, 139, 69).AdaptBackColor(),
+                Color.FromArgb(0, 109, 44).AdaptBackColor(),
+                Color.FromArgb(0, 68, 27).AdaptBackColor(),
+            };
+        }
+
+        private List<GitBlameDisplay> CalculateBlameRecencyData()
+        {
+            if (!AppSettings.BlameShowAuthorAvatar)
+            {
+                return null;
+            }
+
+            var gitBlameDisplays = new List<GitBlameDisplay>(_blame.Lines.Count);
+            var blameLines = _blame.Lines;
+
+            var colors = GetGradients();
+            var mostRecentDate = DateTime.Now.Ticks;
+            var lessRecentDate = Math.Min(DateTime.Now.AddYears(3).Ticks, blameLines.Select(l => l.Commit.AuthorTime).Where(d => d != DateTime.MinValue).Min().Ticks);
```

Comment:
```csharp
DateTime.Now.AddYears(3).Ticks
```
Is it `3` or `-3`? At the first glance I would expect `lessRecentDate` to be in the past rather than future.